be2-scala/src/main/scala/ch/epfl/pop/model/network/method/message/data/lao/StateLao.scala

```diff
@@ -0,0 +1,36 @@
+package ch.epfl.pop.model.network.method.message.data.lao
+
+import ch.epfl.pop.model.network.method.message.data.{ActionType, MessageData, ObjectType, Parsable}
+import ch.epfl.pop.model.objects.{Hash, PublicKey, Timestamp, WitnessSignaturePair}
+
+case class StateLao(
```

Comment:
What are the conventions we are following? in `CreateLao` the parameters are all in one line but here each parameter is in one line. We should choose one convention to follow

Reply:
So I ended up creating the new line only if the line is longer than 100 chars as stated in the coding guidelines. We can however change this if you prefer of course! :)

Reply:
Perfect for me